mmv1/products/vertexai/IndexEndpoint.yaml

```diff
@@ -53,14 +53,18 @@ examples:
       network_name: "network-name"
     test_vars_overrides:
       network_name: 'acctest.BootstrapSharedTestNetwork(t, "vertex-ai-index-endpoint")'
+  - !ruby/object:Provider::Terraform::Examples
+    name: "vertex_ai_index_endpoint_with_psc"
+    primary_resource_id: "index_endpoint"
+    test_vars_overrides:
+      network_name: 'acctest.BootstrapSharedTestNetwork(t, "vertex-ai-index-endpoint")'
```

Comment:
It doesn't look like that `network_name` is being used anywhere in the test. Feel free to remove it if it's not needed

Reply:
I confirmed an acceptance test succeeded after removing it. I've deleted it in [1a0f1ea](https://github.com/GoogleCloudPlatform/magic-modules/pull/8851/commits/1a0f1eae4bbf481f9c32a18ccfeeaf17eed019a1).